internal/cli/server/config.go

```diff
@@ -1131,10 +1155,21 @@ func (c *Config) buildEth(stack *node.Node, accountManager *accounts.Manager) (*
 		n.TrieDirtyCache = calcPerc(c.Cache.PercGc)
 		n.NoPrefetch = c.Cache.NoPrefetch
 		n.Preimages = c.Cache.Preimages
-		n.TransactionHistory = c.Cache.TxLookupLimit
 		n.TrieTimeout = c.Cache.TrieTimeout
 		n.TriesInMemory = c.Cache.TriesInMemory
 		n.FilterLogCacheSize = c.Cache.FilterLogCacheSize
+
+		log.Warn("The flag txlookuplimit will be deprecated soon, please use history.transactions instead")
```

Comment:
Shouldn't this warning be only printed when the flag is set?

Reply:
Handled in https://github.com/maticnetwork/bor/pull/1580/commits/6a57d197f3c8c09479c538014a686043f23dad88. Thanks!